README.md

```diff
@@ -16,16 +16,19 @@ After ensuring the .NET Runtime is installed, download the latest Template BPA r
 To preview the rules that come bundled with the Template BPA, explore [the built-in rules](docs/built-in-bpa-rules.md).
 
 ## Using the Template BPA
-The Template BPA is executed via a command line.  Here is the format to invoke it:
+The Template BPA is executed via a command line.  Here are the formats to invoke it:
 
-`TemplateAnalyzer.exe -t <template-path> [-p <parameters-path>]`
+`TemplateAnalyzer.exe analyze-template <template-path> [-p <parameters-path>]`
+
+`TemplateAnalyzer.exe analyze-directory <directory-path>`
 
 ### Input
 The Template BPA accepts the following inputs:
 
 Argument | Description
 --- | ---
-`-t` or `--template-file-path` | The ARM template to analyze
+`<template-path>` | The ARM template to analyze
+`<directory-path>` | The directory to find ARM templates
```

Comment:
Add a that it recursively finds templates in the directory provided.

Reply:
Agreed.  Can be short and simple.